planetarium/lib9c
# Fix World 1~2 Guide Quest's Rewards

Fix World 1~2 Guide Quest's Rewards

How to test:
1. Check Tables
2. Check Items in the quest window

Reviewers:
@Namyujeong @hskim1028
